test/fuzz/gen.ml

```diff
@@ -4,14 +4,14 @@ open Owi.Symbolic
 module S = Type_stack
 module B = Basic
 
-let expr_always_available block loop expr ~locals ~stack env =
+let expr_always_available block expr ~locals ~stack env =
   [ pair B.const_i32 (const [ S.Push (Num_type I32) ])
   ; pair B.const_i64 (const [ S.Push (Num_type I64) ])
   ; pair B.const_f32 (const [ S.Push (Num_type F32) ])
   ; pair B.const_f64 (const [ S.Push (Num_type F64) ])
   ; pair (const Nop) (const [ S.Nothing ])
-  ; block expr ~locals ~stack env
-  ; loop expr ~locals ~stack env
+  ; block Env.Block expr ~locals ~stack env
+  ; block Env.Loop expr ~locals ~stack env
```

Comment:
You should have only one call to `block` and choose the kind inside the `block` function.